config/cross_section_config.py

```diff
@@ -672,13 +672,13 @@ def __fill_defaults_13TeV__( self ):
             "ttbarRap" : 236.448941265,
         }
         self.tau_values_combined = {
-"MET" : 0.849753435909,
-"WPT" : 1.0235310219,
-"NJets" : 0.932603346883,
+"MET" : 2.15443469003,
+"WPT" : 1.12332403298,
+"NJets" : 1.12332403298,
 "abs_lepton_eta" : 0.1,
-"ST" : 0.932603346883,
+"ST" : 0.642807311728,
 "lepton_pt" : 0.585702081806,
-"HT" : 1.12332403298,
+"HT" : 0.585702081806,
 
```

Comment:
Are the tau values for HT and lepton pt really identical?  Same for WPT and NJets?